ai/app/models/bertcnn.py

```
@@ -0,0 +1,51 @@
+        kernel_size=3,
+        pretrained_model_name="hun3359/klue-bert-base-sentiment",
+        **kwargs
+    ):
+        super().__init__(**kwargs)
+        self.num_labels = num_labels
+        self.hidden_size = hidden_size
+        self.conv_output_channels = conv_output_channels
+        self.kernel_size = kernel_size
+        self.pretrained_model_name = pretrained_model_name
+
+class BertCNNForMultiLabel(PreTrainedModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)